eng/Versions.props

```diff
@@ -225,6 +225,7 @@
     <SystemValueTupleVersion>4.5.0</SystemValueTupleVersion>
     <SystemThreadingTasksExtensionsVersion>4.5.4</SystemThreadingTasksExtensionsVersion>
     <SQLitePCLRawbundle_greenVersion>2.0.4</SQLitePCLRawbundle_greenVersion>
+    <SQLitePCLRawproviderdynamic_cdeclVersion>2.0.4</SQLitePCLRawproviderdynamic_cdeclVersion>
```

Comment:
Is this supposed to be referenced somewhere?

Reply:
@jasonmalinowski ?

Reply:
Yeah this isn't the right change to make, you need to edit the NuGetPackageToIncludeInVsix stuff I linked to in our private conversation.